app/editor/src/features/content/list-view/utils/makeUserFilter.ts

```diff
@@ -0,0 +1,28 @@
+import { IUserFilter } from 'hooks';
+
+export const makeUserFilter = (filter: IUserFilter) => {
```

Comment:
It doesn't really make sense to pass an `IUserFilter` and return ` IUserFilter`.